package.json

```diff
@@ -38,21 +37,18 @@
     "jest": "^29.5.0",
     "prettier": "^2.8.8",
     "ts-jest": "^29.0.5",
-    "ts-node": "^10.9.1",
-    "typedoc": "^0.24.8",
-    "typescript": "^4.9.4",
-    "unique-names-generator": "^4.7.1"
+    "typedoc": "^0.24.8"
   },
   "types": "dist/index.d.ts",
   "files": [
     "/dist"
   ],
   "dependencies": {
-    "@edge-runtime/jest-environment": "^2.3.3",
     "@edge-runtime/types": "^2.2.3",
-    "@sinclair/typebox": "^0.28.15",
-    "@types/web": "^0.0.99",
+    "@sinclair/typebox": "^0.29.0",
```

Comment:
Typebox got a small bump to resolve an error on TS >5.1. I didn't bump all the way to latest because that caused problems with some major releases of TS 4.x.